modules/books.js

```diff
@@ -4,14 +4,14 @@ const mongoose = require('mongoose');
 const { Schema } = mongoose;
 
 const bookSchema = new Schema({
-  Books:Array,
+  Books: Array,
 })
 
 const booksModel = mongoose.model('books', bookSchema);
```

Comment:
Nice work so far on the lab, you guys are well on your way! 💯 

- The Mongoose object has method called model() that takes two args (the first being the collection and the second is the schema)
- The first arg will be singular and uppercase
`const bookModel = mongoose.model('Book', bookSchema)`;

[constructing documents Mongoose](https://mongoosejs.com/docs/models.html)